model/registry/ios.yaml

```
@@ -0,0 +1,44 @@
+groups:
+  - id: registry.ios.lifecycle.events
+    prefix: ios
+    type: attribute_group
+    brief: >
+        The iOS platform on which the iOS application is running.
+    attributes:
+      - id: state
+        stability: experimental
+        note: >
+          The iOS lifecycle states are defined in the [UIApplicationDelegate documentation](https://developer.apple.com/documentation/uikit/uiapplicationdelegate#1656902),
+          and from which the `OS terminology` column values are derived.
+        brief: >
+          This attribute represents the state the application has transitioned into at the occurrence of the event.
+        type:
+          allow_custom_values: false
```

nit: 
```suggestion
          allow_custom_values: true
```

https://github.com/open-telemetry/build-tools/issues/294

---

Fixed!